components/CatalogSearch.vue

```diff
@@ -95,6 +108,7 @@
         &__pill {
             margin-top: $spacer-2;
             cursor: pointer;
+            $font-size-phone: 4;
```

Comment:
```suggestion
```

Reply:
i think this line is necessary to update the selected category. The app breaks without it :0